src/api/store/action.py

```diff
@@ -77,8 +78,8 @@ def create_action(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIEr
         community = Community.objects.get(id=community_id)
         new_action.community = community
       
-      if image:
-        media = Media.objects.create(name=f"{args['title']}-Action-Image", file=image)
+      if image: #now, images will always come as an array of ids 
+        media = Media.objects.filter(pk = image[0]).first()
```

Comment:
how are we ensuring backward compatibility?

Reply:
We are not! I never touched the old uploader on the frontend side. I just added the media library as a new field. So, devs can still create forms with the older uploader, or the media library. And when they do, they will need to handle things on the backend in a way that matches the outputs of the fields... I just thought there is  never going to be a scenario where we are expecting one field to come as the kind of data we are expecting from the media library(array of ids) or the old uploader(file Object) -- its always going to be one way, and it will be one we are prepared for isnt it?